.travis.yml

```diff
@@ -6,6 +6,7 @@ services:
 - docker
 before_install:
 - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+- echo -e "machine github.com\n  login juanesarango \npassword $GH_TOKEN" > ~/.netrc
```

Comment:
should we change the login here from your user to an env variable? `$GH_USERNAME` perhaps?

Reply:
Good idea